rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/license/LicenseController.java

```diff
@@ -138,8 +139,15 @@ public ResponseEntity<CollectionModel<EntityModel<Obligation>>> getObligationsBy
             tags = {"Licenses"}
     )
     @RequestMapping(value = LICENSE_TYPES_URL, method = RequestMethod.GET)
-    public ResponseEntity<CollectionModel<EntityModel<LicenseType>>> getLicenseTypes() throws TException {
-        List<LicenseType> sw360LicenseTypes = licenseService.getLicenseTypes();
+    public ResponseEntity<CollectionModel<EntityModel<LicenseType>>> getLicenseTypes(
+            @RequestParam(value = "search", required = false) String searchElem) throws TException {
```

Comment:
Please add documentation for the parameter with `@Parameter` annotation.

Reply:
done.